tests/unit/generate_detect_target_contour.py

```diff
@@ -0,0 +1,187 @@
+"""
+Helper functions for `test_detect_target_contour.py`.
+
+"""
+
+import cv2
+import math
+import numpy as np
+
+
+LANDING_PAD_COLOUR_BLUE = (100, 50, 50)  # BGR
+
+
+# Test functions use test fixture signature names and access class privates
+# No enable
+# pylint: disable=protected-access,redefined-outer-name
+
+
+class LandingPadImageConfig:
+    def __init__(
+        self,
+        center: tuple[int, int],
+        axis: tuple[int, int],
+        blur: bool,
+        angle: float,
+    ):
+        """
+        Represents the data required to define and generate a landing pad.
+
+        center: The (x, y) coordinates representing the center of the landing pad.
+        axis: The pixel lengths of the semi-major and semi-minor axes of the ellipse.
+        blur: Indicates whether the landing pad should have a blur effect. default: False.
+        angle: The rotation angle of the landing pad in degrees clockwise (0 < angle < 360).
+        """
+        self.center = center
+        self.axis = axis
+        self.blur = blur
+        self.angle = angle
+
+
+class NumpyImage:
+    def __init__(self, image: np.ndarray):
+        """
+        Holds the Numpy Array which represents an image.
+        """
+        self.image = image
+
+
+class BoundingBox:
+    def __init__(self, top_left: tuple[int, int], bottom_right: tuple[int, int]):
+        """
+        Holds the data that define the generated bounding boxes.
+
+        Attributes:
+            top_left: A tuple of integers that represents top left corner of bounding box.
+            bottom_right: A tuple of integers that represents bottom right corner of bounding box.
```

Comment:
`top_left: x, y coordinates representing the top left corner of the bounding box on an image.`

Similar for `bottom_right` .